### ISSUE TITLE
CompileError() with Powershell

### ISSUE THREAD

**Issue by oguzhanogreden** [python-discord/django-simple-bulma#16]
Hi,

I'm getting the following error with Powershell. This is likely because of platform insensitive paths in the finder code.
 
```
File "C:\Users\<username>\envs\djenv\lib\site-packages\sass.py", line 709, in compile
    raise CompileError(v)
       jango_simple_bulma/bulma.sass.ort not found or unreadable: C:Users<username>nvs
        on line 6 of stdin
>> @import "C:\Users\<username>\envs\djenv\Lib\site-packages\django_simple_bulma/bulm
```

Best,
Oguzhan

**Comment by heavysaturn** [comment]
pushing a fix for this now, should be live in 20 minutes.

**Comment by heavysaturn** [comment]
live now on v1.1.4 at https://pypi.org/project/django-simple-bulma/

@oguzhanogreden could you please confirm that this solves the issue? I'm too lazy to boot into windows. :smile:

**Comment by oguzhanogreden** [comment]
This didn't help. It seems my suggested reason was misleading. I'll be making a PR shortly.

**Comment by heavysaturn** [comment]
Solved in release 1.1.5.
